superbuild/projects_modules/qwt.cmake

```diff
@@ -24,7 +24,8 @@ if (NOT USE_SYSTEM_${ep})
 ## Define repository where get the sources
 ## #############################################################################
 
-set(svn_url https://svn.code.sf.net/p/qwt/code/trunk)
+set(git_url ${GITHUB_PREFIX}nebatmusic/qwt.git)
```

Comment:
Will be better on Inria-Asclepios or  FBU-IHU-LIRYC, wont it?

Reply:
You use git repo from unknown guy (who make a fork) and you're afraid to use a repo from one of your collegue :) . 
I Will transfert ownership to asclepios when it will be ok to do it.